### PR TITLE
testing/criu: upgrade to 3.8.1

### PR DESCRIPTION
**Pull request by prspkt**

### CODE REVIEW

File: testing/criu/APKBUILD [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 license="GPL-2.0-or-later LGPL-2.1-only"
-makedepends="xmlto asciidoc python protobuf-c-dev protobuf-dev libaio-dev libnl3-dev
-	py-protobuf libcap-dev perl libnet-dev"
+makedepends="asciidoc libaio-dev libcap-dev libnet-dev libnl3-dev
+	perl protobuf-c-dev protobuf-dev py-protobuf python xmlto"
 checkdepends="py-yaml"
 options="!check" # https://github.com/checkpoint-restore/criu/issues/434
 subpackages="$pkgname-dev $pkgname-doc py-$pkgname:_py:noarch"
-source="$pkgname-$pkgver.tar.gz::https://github.com/xemul/$pkgname/archive/v$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/xemul/criu/archive/v$pkgver.tar.gz"
```
Comment (andypost) [inline testing/criu/APKBUILD]: any reason to change this line?

Comment (prspkt) [reply testing/criu/APKBUILD]: I assume it is easier to copy-paste from source. Please, look at this related [comment](https://github.com/alpinelinux/aports/pull/3648) by @ncopa.

Comment (tmpfile) [reply testing/criu/APKBUILD]: We definitely need a **best practices** document detailing such details.
- Should I use $pkgname on source="..." or the literal package name?
- Should I sed or should I patch?
and all other points of confusion

**Comment by andypost** [comment]
Also tests still fail, commented in https://github.com/checkpoint-restore/criu/issues/434#issuecomment-377701085

**Comment by andypost** [comment]
Filed update https://github.com/alpinelinux/aports/pull/3852

**Comment by prspkt** [comment]
Same with #3852. This is strange since we already run tests as super-user.

**Pull request closed without merging**